prettier.config.cjs

```diff
@@ -1,11 +1,12 @@
 /**
- * Copyright 2021 Google LLC
+ * Copyright 2021 Google LLC.
+ * Copyright (c) Microsoft Corporation.
```

Comment:
The two different styles here end up looking funny, but I guess that's what we get when we each take the form our lawyers prefer. That's fine, let's live with it.